src/System.Management.Automation/engine/Utils.cs

```diff
@@ -616,13 +621,18 @@ internal static string GetRegistryConfigurationPath(string shellID)
             return GetRegistryConfigurationPrefix() + "\\" + shellID;
         }
 
-        // Retrieves group policy settings based on the preference order provided:
-        // Dictionary<string, object> settings = GetGroupPolicySetting("Transcription", Registry.LocalMachine, Registry.CurrentUser);
-
+        // Calling static members of 'Registry' on UNIX will raise 'PlatformNotSupportedException' 
```

Comment:
Yes, this is an example where I think it might raise confusion, so I added comments.